src/childHealth/utils.py

```
@@ -0,0 +1,44 @@
+    plt.scatter(y_test, y_pred, alpha=0.5)
+    plt.plot([y_test.min(), y_test.max()], [y_test.min(), y_test.max()], 'r--', lw=2)
+    plt.xlabel("Actual Price")
+    plt.ylabel("Predicted Price")
+    plt.title("Actual vs Predicted House Prices")
+    plt.tight_layout()
+    plt.show()
+
+def plot_feature_importance(feature_importance, feature_names, top_n=10):
+    """
+    Plot the top N feature importances.
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)